tests/integration/buildcmd/test_build_cmd_arm64.py

```diff
@@ -0,0 +1,457 @@
+                {"app.js", "app.js.map"},
+                "app.lambdaHandler",
+                "use_container",
+                "arm64",
+            ),
+        ]
+    )
+    def test_building_default_package_json(
+        self, runtime, code_uri, expected_files, handler, use_container, architecture
+    ):
+        self._test_with_default_package_json(runtime, use_container, code_uri, expected_files, handler, architecture)
+
+
+class TestBuildCommand_EsbuildFunctions_With_External_Manifest_arm64(BuildIntegEsbuildBase):
+    template = "template_with_metadata_esbuild.yaml"
+    MANIFEST_PATH = "Esbuild/npm_manifest/package.json"
+
+    @parameterized.expand(
+        [
+            (
+                "nodejs16.x",
+                "Esbuild/Node_without_manifest",
+                {"main.js", "main.js.map"},
+                "main.lambdaHandler",
+                False,
+                "arm64",
+            ),
+            (
+                "nodejs18.x",
+                "Esbuild/Node_without_manifest",
+                {"main.js", "main.js.map"},
+                "main.lambdaHandler",
+                False,
+                "arm64",
+            ),
+            (
+                "nodejs16.x",
+                "Esbuild/TypeScript_without_manifest",
+                {"app.js", "app.js.map"},
+                "app.lambdaHandler",
+                False,
+                "arm64",
+            ),
+            (
+                "nodejs18.x",
+                "Esbuild/TypeScript_without_manifest",
+                {"app.js", "app.js.map"},
+                "app.lambdaHandler",
+                False,
+                "arm64",
+            ),
+        ]
+    )
+    def test_building_default_package_json(
+        self, runtime, code_uri, expected_files, handler, use_container, architecture
+    ):
+        self._test_with_default_package_json(runtime, use_container, code_uri, expected_files, handler, architecture)
+
+
+class TestBuildCommand_NodeFunctions_With_Specified_Architecture_arm64(BuildIntegNodeBase):
+    template = "template_with_architecture.yaml"
+
+    @parameterized.expand(
+        [
+            ("nodejs12.x", False, "arm64"),
+            ("nodejs14.x", False, "arm64"),
+            ("nodejs16.x", False, "arm64"),
+            ("nodejs18.x", False, "arm64"),
+            ("nodejs12.x", "use_container", "arm64"),
+            ("nodejs14.x", "use_container", "arm64"),
+            ("nodejs16.x", "use_container", "arm64"),
+            ("nodejs18.x", "use_container", "arm64"),
+        ]
+    )
+    def test_building_default_package_json(self, runtime, use_container, architecture):
+        self._test_with_default_package_json(runtime, use_container, self.test_data_path, architecture)
+
+
+class TestBuildCommand_RubyFunctions_With_Architecture_arm64(BuildIntegRubyBase):
+    template = "template_with_architecture.yaml"
+
+    @parameterized.expand([("ruby2.7", "arm64", "Ruby"), ("ruby3.2", "arm64", "Ruby32")])
```

Comment:
nit: Do we need to parameterize the architecture if these are all arm64 (these test cases and others in this file)?